test/cli/help/main/expected-stdout

```diff
@@ -34,6 +34,7 @@ Available Commands:
   local validate                 Validate the local directory.
   local fix-paths                Ensure that all paths match configured naming.
   local template                 Manage templates in the local directory.
+  local template delete          Delete template instance.
```

Comment:
Na delete, by bolo dobre napisat E2E test, kludne v dalsom PR.

Reply:
Jasně, to mám v plánu jak dám dohromady ty zbývající kousíčky.